__main__ -- ERROR -- test message: 36
__main__ -- ERROR -- test message: 37
